### DISCUSSION TITLE
Docker创建改变端口

### DISCUSSION THREAD

**Discussion by xzl0326** [messense/aliyundrive-webdav#652] [Q&A]
请问Docker创建的是不能改变端口吗？我只要改了端口就打不开了

**Comment by lanyucomputer** [comment]
docker run -d --name=aliyun --restart=unless-stopped -p 4444:8080 \
  -v /etc/aliyundrive-webdav/:/etc/aliyundrive-webdav/ \
  -e REFRESH_TOKEN='' \
  -e WEBDAV_AUTH_USER=admin \
  -e WEBDAV_AUTH_PASSWORD=admin \
  messense/aliyundrive-webdav

**Comment by lanyucomputer** [comment]
4444这里改成你想要的端口

**Reply by xzl0326** [reply]
好的。谢谢

**Comment by lanyucomputer** [comment]
docker run -d --name=aliyunzhang --restart=unless-stopped \
  -v /etc/aliyundrive-webdav/:/etc/aliyundrive-webdav/ \
  -p <你想改的端口>:8080 \
  -e REFRESH_TOKEN='yourtoken' \
  -e WEBDAV_AUTH_USER=admin \
  -e WEBDAV_AUTH_PASSWORD=admin \
  messense/aliyundrive-webdav \
  /usr/bin/aliyundrive-webdav --host 0.0.0.0 --auto-index --workdir /etc/aliyundrive-webdav --port 8080 --root /<网盘目录路径>
